tests/acceptance/features/apiSpaces/search.feature

```diff
@@ -121,7 +120,6 @@ Feature: Search
     And the search result should contain "1" entries
     And the search result of user "Brian" should contain these entries:
       | /<searchObject> |
-    And for user "Brian" the response should contains the parent "<parentFolder>" from mountpoint "folderMain"
```

Comment:
don't we need that check anymore?

Reply:
I deleted case. It is duplicate with `report.feature`